Source/Tools/FEXServer/FDCountWatch.cpp

```diff
@@ -0,0 +1,76 @@
+#include <atomic>
+#include <cstddef>
+#include <cstdint>
+#include <cstring>
+#include <filesystem>
+#include <mutex>
+#include <sys/resource.h>
+
+namespace FDCountWatch {
+  // FD count watching
+  constexpr size_t static MAX_FD_DISTANCE = 32;
+  rlimit MaxFDs{};
+  std::atomic<ssize_t> NumFilesOpened{};
+  std::mutex FDLimitMutex;
```

Comment:
These should probably be at least static.

Reply:
Switched to static.